### PR TITLE
Metatags added for homepage

### LINKED ISSUE #3212: Preview error's for Facebook and Reddit
Pasting a link on Facebook to create a post shows multiple previews/ previews without image even if the book has a book cover.
Reddit previews don't show up.

### Evidence / Screenshot (if possible)
Screenshot from 2020-03-19 17-15-05
Screenshot from 2020-03-19 17-09-20
Screenshot from 2020-03-15 16-57-13
Screenshot from 2020-03-15 16-56-44

### Steps to Reproduce

1. Go to facebook/reddit
2. click on create a post
3. Paste homepage/edition/work link to share

### Stakeholders

@tabshaikh

### PR DESCRIPTION
**Pull request by tabshaikh**
Closes #3212 

[fix] 

### Technical

Closing the above issue for the homepage I would open followup issues for the images not showing up on the author and editions page 

### Screenshot

Please Ignore the content in the screenshots :stuck_out_tongue: . Just there to see that the image is showing up :)
Screenshot from 2020-05-12 06-12-29
Screenshot from 2020-05-12 06-09-35

### Stakeholders

@mekarpeles

### CODE REVIEW

**Comment by tabshaikh** [comment]
@mekarpeles This line would work as leaving the title contents empty google will take the sites title as the content of meta title tag https://github.com/internetarchive/openlibrary/blob/4ac311d1e964e22241a713454e6ae4c3f2433c1a/openlibrary/templates/site/head.html#L7
https://www.quora.com/If-I-leave-meta-description-of-my-post-blank-will-Google-automatically-create-a-meta-description-for-that-particular-post-I-use-Yoast-SEO-plugin

File: openlibrary/templates/home/index.html [hunk 1]
```diff
@@ -3,6 +3,18 @@
 $ _x = ctx.setdefault('bodyid', 'home')
 $var title: $_("Welcome to Open Library")
 
+$ meta_description = "Open Library is an open, editable library catalog, building towards a web page for every book ever published."
+$ meta_image_url = "https://openlibrary.org/static/images/ol-banner.png"
```
Comment (mekarpeles) [inline openlibrary/templates/home/index.html:7]: This still needs the ol-banner.png included

Comment (tabshaikh) [reply openlibrary/templates/home/index.html:7]: OL banner added

**Comment by Callie8909** [comment]
thanks

**Pull request merged**